styles/styles.css

```diff
@@ -0,0 +1,202 @@
+  animation: fadeIn ease-in 1;
+
+  -webkit-animation-fill-mode: forwards;  /* this makes sure that after animation is done we remain at the last keyframe value (opacity: 1)*/
+  -moz-animation-fill-mode: forwards;
+  animation-fill-mode: forwards;
+
+  -webkit-animation-duration: 1s;
+  -moz-animation-duration: 1s;
+  animation-duration: 3s;
+}
+
+/* Divider Line */
+.quote::after, .sponsors::after {
+  display: block;
+  content: "";
+  border-top: 1px dashed #426642;
+  margin-top: 2em;
+}
+
+/* Top Nav Bar*/
+nav {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  background-color: black;
+  color: whitesmoke;
+}
+
+nav ul {
+  display: grid;
+  flex-direction: row;
+  grid-template-columns: 5fr 1fr 1fr;
+}
+
+/* Intro Section */
+.intro {
+  display: flex
+  background-color: lightgrey;
+  padding: 2em;
+  /* Parallax Effect */
+  background-image: url("https://cdn.pixabay.com/photo/2016/09/30/03/39/pineapple-1704339_1280.jpg");
+  /* Set a specific height */
+  height: 500px; 
+  /* Create the parallax scrolling effect */
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+sub {
+  display: block;
+  margin-bottom: 3em;
+}
+
+.intro [href] {
+  width: 25%;
+  border: .25em solid #426642;
+  border-radius: 5px;
+  align: center;
+  padding: .5em;
+}
+
+/* Quote Section */
+.quote {
+  font-style: italic;
+}
+
+blockquote::before, blockquote::after {
+  content: '"';
+}
+
+.quote p {
+  text-align: end;
+  margin-right: 5em;
+}
+
+.quote p::before {
+  content: "--";
+}
+
+/* Why Section */
+.why {
+  /* Parallax Effect */
+  background-image: url("https://cdn.pixabay.com/photo/2016/11/22/23/15/abstract-1851115_1280.jpg");
+  /* Set a specific height */
+  height: 300px; 
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.why ul {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 2em;
+}
+
+.why img {
+  float: left;
+  padding-right: .5em;
+}
+
+/* Team Section */
+.team {
+  color: white;
+  /* Parallax Effect */
+  background-image: url("https://cdn.pixabay.com/photo/2017/07/25/14/54/rain-2538429_1280.jpg");
+  /* Set a specific height */
+  height: 700px; 
+  /* Create the parallax scrolling effect */
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+.team ul {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  padding: 1em;
+}
+
+.team img {
+  display: block;
+  border-radius: 50%;
+  margin: 1em 7em;
+}
+
+.team span {
+  display: block;
+}
+
+/* Sponsors Section */
+.sponsors ul {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  padding: .5em;
+}
+
+.sponsors li {
+  align-self: center;
+  margin: .5em;
+}
+
+/* Footer Section */
+.footer {
+  display: flex;
+}
+
+.footer div {
+  padding: 0 10em 0 2em;
```

Comment:
I like how you combine the padding measurements in the one property. I think I had a few instances where I used "padding-top" and "padding-bottom" instead of just combining them.